src/devices/acurite.c

```diff
@@ -376,26 +389,13 @@ static int acurite_6045_decode(r_device *decoder, bitbuffer_t *bitbuffer, unsign
     *rawp = '\0';
 
     // Flag whether this message might need further analysis
-    if ((message_type != ACURITE_MSGTYPE_6045M) // 6045 message type is 0x2f
```

Comment:
Why remove the `message_type` check? Is it incorrect?

Reply:
When that was originally written, devices were differentiated by message length.

I thought the casing for device had all been changed over to message_type, but I see now that I'm wrong. 

I'll fix that now.